zcash_primitives/CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this library adheres to Rust's notion of
   - `zcash_primitives::prover`
   - `zcash_primitives::redjubjub`
   - `zcash_primitives::util::{hash_to_scalar, generate_random_rseed}`
+- Renamed the following in `zcash_primitives::transaction::components` to use
+  lower-case abbreviations (matching Rust naming conventions):
+  - `JSDescription` to `JsDescription`
+  - `SproutProof::PHGR` to `SproutProof::Phgr`
```

Comment:
Oof, sometimes I really dislike this convention.

Reply:
Yeah, an acronym is not an abbreviation.

Reply:
Rust uses this convention for acronyms as well (e.g. `Html` instead of `HTML`); I'm using "abbreviation" as a catch-all. If there are specific cases where we want to break from that convention, we could consider them on a case-by-case basis, but our long-standing precedent is to follow Rust style convention.

Reply:
Easier just to accept what Clippy is telling us, I guess. :)

Reply:
It's the fact that these are people's names (Jon Howell, Craig Gentry, and Mariana Raykova) getting lowercased that is bothering me. Not that there's anything wrong with lowercase names, but only if the person spells them that way. Can I suggest just suppressing the lint for this case?

Reply:
ACK, I'll revert this specific change.